web/src/features/exchanges/ExchangeArrow.tsx

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable unicorn/no-null */
+import { useMemo } from 'react';
+import { MapboxMap } from 'react-map-gl';
+import { resolvePath } from 'react-router-dom';
+import { ExchangeArrowData } from 'types';
+import { quantizedCo2IntensityScale, quantizedExchangeSpeedScale } from 'utils/scales';
+
+interface ExchangeArrowProps {
+  data: ExchangeArrowData;
+  viewportWidth: number;
+  viewportHeight: number;
+  map: MapboxMap;
+}
+
+function ExchangeArrow({ data, viewportWidth, viewportHeight, map }: ExchangeArrowProps) {
+  const mapZoom = map.getZoom();
+  const colorBlindModeEnabled = false; // TODO: FIX
+  const absFlow = Math.abs(data.netFlow || 0);
```

Comment:
Should this use nullish coalescing instead?
```suggestion
  const absFlow = Math.abs(data.netFlow ?? 0);
```

Reply:
Yes :)